packages/@sfcc-bff/basketapi/src/api/schema/basketTypeDef.ts

```diff
@@ -63,4 +66,11 @@ export const basketTypeDef = gql`
         itemText: String
         price: Float
     }
+
+    type couponItem {
+        valid: Boolean
+        code: String
+        couponItemId: String
```

Comment:
Should `couponItemId` and `code` be required?

Reply:
Good point, fixing it.